Part A - Basics/07_control_flow/examples/control_with_break_statement.py

```diff
@@ -9,13 +9,12 @@
 
 
 # Example 2: Break statement inside while loop
-val = input("Enter a number: ")
```

Comment:
**Rejected**

It makes the code more unreadable and it doesn't protect the program anyhow.